taproot-updates.md

```diff
@@ -195,7 +194,7 @@ A taproot HTLC-success transaction looks like:
       "txid": "...",
       "vout": 42,
       "scriptSig": "<payment_preimage> <remotehtlcsig> <localhtlcsig>",
-      "sequence": 0
+      "sequence": 1
```

Comment:
also without https://github.com/t-bast/lightning-docs/pull/19/files#r869579106, the sequence needs to be 1